whatsapp_bot.py

```diff
@@ -7,16 +7,17 @@
 
 from utils import query_vectara
 
-account_sid = os.getenv("ACCOUNT_SID")
-auth_token = os.getenv("Auth_TOKEN")
```

Comment:
Nit: why is ACCOUNT_SID all upper case whereas the Auth_TOKEN half and half? does it make sense to make those consistent?